.azuredevops/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## Proposed Changes
+What kind of change does this pull request introduce?   
+
+[comment]:# (Please check the ones that apply.)
+
+- [ ] Feature
+- [ ] Bugfix
+- [ ] Code style update (formatting)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Pipeline related changes
+- [ ] Documentation content changes
+- [ ] Other... Please describe:     
+
+## Description
```

Comment:
shouldn't the pr template be similar to the oneapptemplate one https://github.com/nventive/UnoApplicationTemplate/blob/main/.github/PULL_REQUEST_TEMPLATE.md?plain=1

- Missing versioning impact
- Missing always applicable vs. contexual check

Reply:
nvm I mixed up this one with the other one

Reply:
I copied it from the actual repository. May be I made a mistake?